lib/manager/components/deployment/PeliasPanel.js

```diff
@@ -23,43 +23,30 @@ type Props = {
   project: Project,
   setErrorMessage: typeof setErrorMessage,
   updateDeployment: typeof updateDeployment,
+  updatePelias: typeof updatePelias,
   uploadPeliasWebhookCsvFile: typeof uploadPeliasWebhookCsvFile
 }
 
 type State = {
   fileToDeleteOnSuccesfulUpload: string | null,
-  peliasWebhookUrl: string
 }
 
 class PeliasPanel extends Component<Props, State> {
   state = {
-    fileToDeleteOnSuccesfulUpload: null,
-    peliasWebhookUrl: ''
+    fileToDeleteOnSuccesfulUpload: null
   };
 
-  componentDidMount = () => {
-    const { deployment } = this.props
-    this.setState({
-    // Auto-populate blank field if project has webhook URL in it
-      peliasWebhookUrl: deployment.peliasWebhookUrl || ''
-    })
-  };
-
-  /**
-   *  Method fired when Pelias Update checkbox is checked
-   */
-  _onChangeUpdatePelias = () =>
-    this._updateDeployment({
-      peliasUpdate: !this.props.deployment.peliasUpdate
-    });
-
+  _onClickPeliasUpdate = () => {
+    const {deployment, updatePelias} = this.props
+    updatePelias(deployment)
+  }
   /**
    *  Method fired when Pelias Update checkbox is checked
    */
```

Comment:
Is this right? This function doesn't appear to be connected to a checkbox.

Reply:
Good eye again! Reworded in f6d26732